sdk/schemaregistry/azure-schemaregistry/samples/async_samples/sample_code_schemaregistry_async.py

```diff
@@ -120,8 +133,9 @@ async def get_schema_id(schema_registry_client):
 async def main():
     client, credential = create_client()
     async with client, credential:
-        schema_id = await register_schema(client)
-        schema = await get_schema(client, schema_id)
+        schema_properties = await register_schema(client)
+        schema = await get_schema(client, schema_properties.id)
+        schema = await get_schema_by_version(client, schema_properties.version)
```

Comment:
I am thinking a sample should show a champion scenario, maybe consider adding a stand alone one that registers multiple versions of a schema and retrieves an older version?

Reply:
updated to include methods that register multiple schemas and grab the older one